webpack/components/AnsibleHostDetail/components/AnsibleVariableOverrides/AnsibleVariableOverridesTableHelper.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import TimesIcon from '@patternfly/react-icons/dist/js/icons/times-icon';
+import CheckIcon from '@patternfly/react-icons/dist/js/icons/check-icon';
+import { Link } from 'react-router-dom';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+const formatSourceLink = (hostAttrs, { currentValue }) => {
+  const value = `${currentValue.element}: ${currentValue.elementName}`;
+  let { element } = currentValue;
+  const { meta } = currentValue;
+
+  const knownMatchResources = ['os', 'hostgroup', 'domain'];
+
+  if (!knownMatchResources.includes(element)) {
+    return value;
+  }
+
+  if (element === 'os') {
+    element = 'operatingsystem';
+  }
+
+  const elementId = hostAttrs[`${element}_id`];
+
+  if (elementId && meta?.canEdit) {
+    return <Link to={`/${element}s/${elementId}/edit`}>{value}</Link>;
```

Comment:
if the user can't edit it, we can make it disabled rather than removing it?
@MariSvirik any thoughts?

Reply:
I return value without link 3 lines below.